src/screens/FeedScreen/hooks/useContentFeed.js

```diff
@@ -2,8 +2,8 @@
 import React from 'react';
 import {Text} from 'react-native';
 import reactStringReplace from 'react-string-replace';
-import {colors} from '../../../utils/colors';
 import {getUserId} from '../../../utils/token';
+import {COLORS} from '../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to import `COLORS` from the `utils/theme` module instead of importing `colors` from the `utils/colors` module. This change seems to be part of a larger refactoring effort to standardize color usage across the application. As long as the `COLORS` object in the `utils/theme` module contains all the necessary color constants, this change should not introduce any issues.
```diff
- import {colors} from '../../../utils/colors';
+ import {COLORS} from '../../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->